modules/auth/sso/reverseproxy.go

```diff
@@ -79,7 +79,7 @@ func (r *ReverseProxy) VerifyAuthData(req *http.Request, w http.ResponseWriter,
 
 	// Make sure requests to API paths, attachment downloads, git and LFS do not create a new session
 	if !middleware.IsAPIPath(req) && !isAttachmentDownload(req) && !isGitRawOrLFSPath(req) {
-		if sess.Get("uid").(int64) != user.ID {
+		if sess != nil && (sess.Get("uid") != nil || sess.Get("uid").(int64) != user.ID) {
```

Comment:
`sess.Get("uid") == nil`?

Reply:
Agh! I think the force push must have failed! Thanks for spotting. Fixed!